jujugui/static/gui/src/app/components/profile/profile.js

```diff
@@ -164,19 +165,15 @@ class Profile extends React.Component {
       sectionsMap.set('invoices', {
         label: 'Invoices',
         getComponent: () => {
-          return (
-            <ProfileInvoiceList
-              baseURL={props.baseURL}
-              user={props.userInfo.external} />);
+          return (<ProfileInvoiceList baseURL={props.baseURL}
```

Comment:
I think the old layout for this was easier to read

Reply:
Perhaps but bear in mind if we make the max props 0 - you'll also get;

```
<div
	className="foo">
Baz
</div>
```

I think it looks a little off on first glance because you're not used to it - give it a week and you'll forget it was ever anything else.

Reply:
looking at the eslint rule it looks like it should have allowed the original layout so I'm not sure why the change.

Reply:
@hatched Prettier will run and inline all props/attributes until the character limit before breaking onto a new line - then it will pass off to Eslint where the rule above will kick in and it'll make the change above.